conclave-common/src/main/kotlin/com/r3/conclave/common/internal/attestation/QuoteVerifier.kt

```diff
@@ -445,13 +445,21 @@ object QuoteVerifier {
         return EnclaveTcbStatus.Revoked
     }
 
-    private fun getMatchingTcbLevel(pckExtensions: SGXExtensionASN1Parser, tcbInfo: TcbInfo): TcbStatus {
+    private fun getMatchingTcbLevel(pckExtensions: SGXExtensionASN1Parser, tcbInfo: TcbInfo): TcbLevel {
         val pckTcbs = IntArray(16) { pckExtensions.getInt("${SGX_TCB_OID}.${it + 1}") }
         val pckPceSvn = pckExtensions.getInt(SGX_PCESVN_OID)
 
-        for (lvl in tcbInfo.tcbLevels) {
-            if (isCpuSvnHigherOrEqual(pckTcbs, lvl.tcb) && pckPceSvn >= lvl.tcb.pcesvn) {
-                return lvl.tcbStatus
+        for (tcbLevel in tcbInfo.tcbLevels) {
+            if (isCpuSvnHigherOrEqual(pckTcbs, tcbLevel.tcb) && pckPceSvn >= tcbLevel.tcb.pcesvn) {
+                if (tcbInfo.version.id >= 3 && tcbInfo.id == TypeID.TDX) {
+                    /**
+                     * Ignore TDX quotes for now
+                     * TODO: CON-1273, Audit changes to the intel QVL
+                     */
+                    continue
```

Comment:
throw IllegalArgumentException with a message saying TDX quotes are not supported. We don't want to unwittingly accept an invalid TDX quote.